rotkehlchen/tests/api/test_makerdao_dsr.py

```diff
@@ -677,6 +677,7 @@ def mock_get_logs(
     assert len(errors) == 0
 
 
+@pytest.mark.vcr
```

Comment:
Nope. This queries the history for an account and this history can change. This is not something vcr can or should record

Reply:
Okay as discussed in the call let's go with this. Use vcr anywhere we can and avoid any complicated mocks.